src/core/index.js

```diff
@@ -1,5 +1,5 @@
 export {default as _adapters} from './core.adapters';
-export {default as Animation} from './core.animation';
+export {default as SingleAnimation} from './core.animation';
```

Comment:
What is the reason for the name change?

Reply:
The plugin was also named `Animation`, so it was to avoid a clash. I thought about making the plugin something like `AnimationPlugin`, but none of the other plugins had a `Plugin` suffix. And I thought it'd be better to give the shorter name to the plugin because it's something users would use regularly whereas I think this class is only used in tests and unlikely to be referenced by users.